system-addon/lib/DownloadsManager.jsm

```diff
@@ -52,14 +44,7 @@ this.DownloadsManager = class DownloadsManager {
 
   formatDownload(element) {
     const downloadedItem = element.download;
-    let description;
-    if (element.fileType) {
-      // If we have a file type: '1.5 MB — PNG'
-      description = `${element.sizeStrings.stateLabel} \u2014 ${element.fileType}`;
-    } else {
-      // If we do not have a file type: '1.5 MB'
-      description = `${element.sizeStrings.stateLabel}`;
-    }
+    const description = `${element.sizeStrings.stateLabel}`;
```

Comment:
nit: just get rid of the backtick string and use the value? ;)

Reply:
ehm. oops